### PR TITLE
Feature/lots

### PR DESCRIPTION
**Pull request by cameronblandford**

### CODE REVIEW

File: populate.py [hunk 1]
```diff
@@ -20,25 +20,25 @@
...
+#             u = models.User.objects.get(username=u.username)
+#         items = factories.ItemFactory.build_batch(random.randint(1, 10), collection=c, created_by=u)
+#         for i in items:
+#             print("new item: " + i.title)
+#             i.status = random.choice(
+#                 ['approved', 'approved', 'approved', 'approved', 'pending', 'pending', 'rejected'])
+#             i.date_submitted = datetime.datetime.now(tz=pytz.timezone('US/Eastern'))
+#             i.date_accepted = datetime.datetime.now(tz=pytz.timezone('US/Eastern'))
+#             i.type = 'preprint'
+#             i.save()
```
Comment (NyanHelsing) [inline populate.py]: Is this meant to be commented?

File: workflow/fixtures/workflow_data.json [hunk 2]
```diff
@@ -0,0 +1 @@
...kflow": 1}}, {"model": "workflow.case", "pk": 386, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 387, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 388, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 389, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 390, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 391, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 392, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 393, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 394, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 395, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 396, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 397, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 398, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 399, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 400, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 401, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 402, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 403, "fields": {"workflow": 1}}]
```
Comment (NyanHelsing) [inline workflow/fixtures/workflow_data.json:1]: This fixture is kinda messy. Lets make a ticket to make a clean one

File: populate.py (hunk 1 above)
Comment (cameronblandford) [reply populate.py]: Yeah, for now

File: workflow/fixtures/workflow_data.json (hunk 2 above)
Comment (cameronblandford) [reply workflow/fixtures/workflow_data.json:1]: Sounds good. I think it's worth having this in develop for now though

Comment (NyanHelsing) [reply workflow/fixtures/workflow_data.json:1]: Agreed

File: tests/factories.py [hunk 3]
```diff
@@ -26,7 +26,7 @@ class Meta:
     title = factory.Faker('text', max_nb_chars=75)
     description = factory.Faker('text', max_nb_chars=500)
     status = 'approved'
-    source_id = 'fxsa9'
+    source_id = 'xrfye'
```
Comment (NyanHelsing) [inline tests/factories.py:29]: I've noticed a lot of churn with these guids here. Lets make a ticket to see if theres a way to make this config or make it dynamically.

**Review by NyanHelsing — APPROVED** [review]
Looks great. Merging.

**Pull request merged**